### ISSUE TITLE
Rotation center of Scene Camera lost at save / open

### ISSUE THREAD

**Issue by peteihis** [ArtOfIllusion/ArtOfIllusion#257]
I'm pretty sure that `SceneCamera` was supposed to remember the point it was last rotated around, when you open a saved file, but it seems that it does not. 

Some other issues too, like handling the non-perspective mode seem to be closely related.

I'll need to have look at the code.

**Comment by peteihis** [comment]
And I remember right: 
Line 565: `out.writeDouble(distToPlane);`
Line 529: `distToPlane = in.readDouble();`

The `distToPlane` being the working depth of that view, where the roatation center is supposed to be too. but the information never ends up into the Camera of the view at start-up.

**Comment by peteihis** [comment]
And it gets weirder...

I checked the code. If there is an object camera present then the view uses that working depth directly, so it should not be possible to lose the value at any point of the process. Also did a test and NOW it works as it should. However...

I was changing the SceneCamera from perspective to orthographic mode. That probably had an effect. At some point it also mis-scaled the 3D-manipulator way too big. Not sure in what combination of everything, but it did.

Basically the perpective/parallell toggle on the SceneCamera should be made to follow the logic of the plain view and it's Camera in the handling of it. Unfortunalety there are a few more variables to think about.  Also all the parameters should need to be added to the Pose Track of the SceneCamera too. Now at least the perspective-or-not is not stored there. It's a bit of rebel as a parameter to save, but I suppose I can come up with something reasonable.
